wrapper/src/main/java/software/amazon/jdbc/PropertyDefinition.java

```diff
@@ -48,6 +48,12 @@ public class PropertyDefinition {
       new AwsWrapperProperty(
           "wrapperPlugins", null, "Comma separated list of connection plugin codes");
 
+  public static final AwsWrapperProperty PRESERVE_PLUGINS_ORDER =
```

Comment:
This is an interesting one.
How do we put this plugin order thing in the documentation?
If we want the sorting to be on by default, why not revert the wording from PRESERVE_PLUGINS_ORDER to AUTOMATICALLY_SORT_PLUGINS/AUTO_SORT_PLUGIN_ORDER ?

In this way, we could add something in the docs like this:
"This flag is enabled by default, meaning that the plugins order will be auto generated according to the order that we expect the plugins to be ordered. Disable it at your own risk or if you really need some plugins to be executed in a particular order"